templates/web-ui/payment-form/basic-digital-wallet/sqpaymentform-basic.html

```diff
@@ -17,58 +17,60 @@
 
 <body>
 
-<div id="sq-ccbox">
-  <!--
-    You should replace the action attribute of the form with the path of
-    the URL you want to POST the nonce to (for example, "/process-card")
-  -->
-  <form id="nonce-form" novalidate action="path/to/payment/processing/page" method="post">
-    Pay with a Credit Card
-    <table>
-    <tbody>
-      <tr>
-        <td>Card Number:</td>
-        <td><div id="sq-card-number"></div></td>
-      </tr>
-      <tr>
-        <td>CVV:</td>
-        <td><div id="sq-cvv"></div></td>
-      </tr>
-      <tr>
-        <td>Expiration Date: </td>
-        <td><div id="sq-expiration-date"></div></td>
-      </tr>
-      <tr>
-        <td>Postal Code:</td>
-        <td><div id="sq-postal-code"></div></td>
-      </tr>
-      <tr>
-        <td colspan="2">
-          <button id="sq-creditcard" class="button-credit-card" onclick="requestCardNonce(event)">
-            Pay with card
-          </button>
-        </td>
-      </tr>
-    </tbody>
-    </table>
-
-    <!--
-      After a nonce is generated it will be assigned to this hidden input field.
-    -->
-    <input type="hidden" id="card-nonce" name="nonce">
-  </form>
-</div>
-
-<div id="sq-walletbox">
-  Pay with a Digital Wallet
-  <div id="sq-apple-pay-label" class="wallet-not-enabled">Apple Pay for Web not enabled</div>
-  <!-- Placeholder for Apple Pay for Web button -->
-  <button id="sq-apple-pay" class="button-apple-pay"></button>
-
-  <div id="sq-masterpass-label" class="wallet-not-enabled">Masterpass not enabled</div>
-  <!-- Placeholder for Masterpass button -->
-  <button id="sq-masterpass" class="button-masterpass"></button>
-</div>
+  <div id="form-container">
 
+    <div id="sq-walletbox">
+      <div id="sq-apple-pay-label" class="wallet-not-enabled">Apple Pay for Web not enabled.</div>
+      <!-- Placeholder for Apple Pay for Web button -->
+      <button id="sq-apple-pay"></button>
+
+      <div id="sq-masterpass-label" class="wallet-not-enabled">Masterpass not enabled.</div>
+      <!-- Placeholder for Masterpass button -->
+      <button id="sq-masterpass"></button>
+
+      <div id="sq-walletbox-divider">
+        <span id="sq-walletbox-divider-label">Or</span>
+        <hr />
+      </div>
+    </div>
+
+    <div id="sq-ccbox">
+      <!--
+        You should replace the action attribute of the form with the path of
+        the URL you want to POST the nonce to (for example, "/process-card")
+      -->
+      <form id="nonce-form" novalidate action=".process-card.php" method="post">
```

Comment:
Suggest changing the action parameter back to: `action="path/to/payment/processing/page"` so it's more obvious that the path needs to be updated and we're not implying this template only works with PHP.